app/Repositories/Place/PlaceCategoryRepository.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Hedonist\Repositories\Place;
+
+use Prettus\Repository\Eloquent\BaseRepository;
+use Prettus\Repository\Criteria\RequestCriteria;
+use Prettus\Repository\Contracts\CriteriaInterface;
+use Illuminate\Database\Eloquent\Collection;
+use Hedonist\Entities\Place\PlaceCategory;
+
+
+class PlaceCategoryRepository extends BaseRepository implements PlaceCategoryRepositoryInterface
+{
+    public function save(PlaceCategory $placeCategory): PlaceCategory 
+    {
+        $placeCategory->save();
+        
+        return $placeCategory;
+    }
+    
+    public function getById(int $id): PlaceCategory
+    {
+        return PlaceCategory::findOrFail($id);
```

Comment:
Better to use such method signature `getById(int $id): ?PlaceCategory`. 

`findOrFail` method throws uncatched exception also, use just `find`.